grpc/tests/grpctest.cpp

```diff
@@ -108,15 +104,14 @@ int main(int /*argc*/, const char * /*argv*/[]) {
   auto stub = MyGame::Example::MonsterStorage::NewStub(channel);
 
 
-  flatbuffers::FlatBufferBuilder fbb;
+  flatbuffers::grpc::MessageBuilder fbb;
   {
     grpc::ClientContext context;
     // Build a request with the name set.
     auto monster_offset = CreateMonster(fbb, 0, 0, 0, fbb.CreateString("Fred"));
     fbb.Finish(monster_offset);
-    auto request = flatbuffers::BufferRef<Monster>(fbb.GetBufferPointer(),
-                                                   fbb.GetSize());
-    flatbuffers::BufferRef<Stat> response;
+    auto request = fbb.ReleaseMessage<Monster>();
+    flatbuffers::grpc::Message<Stat> response;
 
     // The actual RPC.
     auto status = stub->Store(&context, request, &response);
```

Comment:
We should check the status here specifically for verifier failure (and only verifier failure, not other occurrences of DATA_LOSS).